src/test-support/mocks/src/meshes/_/meshgateways.ts

```diff
@@ -36,14 +38,27 @@ export default ({ env, fake, pager }: EndpointDependencies): MockResponder => (r
             },
           ],
           conf: {
-            listeners: [
-              {
-                hostname: 'foo.example.com',
-                port: 8080,
-                protocol: 'HTTP',
+            listeners: Array.from({ length: listenerCount }).map(() => {
+              return {
+                hostname: `${fake.internet.domainWord()}.${fake.internet.domainName()}`,
+                port: fake.internet.port(),
+                protocol: fake.helpers.arrayElement<'HTTP' | 'HTTPS' | 'TCP' | 'TLS'>(['HTTP', 'HTTPS', 'TCP', 'TLS']),
```

Comment:
Do we need to specify the types for these things? (I think I saw another one further up also)

Reply:
Yes, the method is otherwise typed as returning an array of strings which fails the expected type.

Reply:
What if we used `as const` on these instead?

Reply:
I don’t see a reason to do that. Although a bit verbose, this gives us what we need here.